.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+---
+version: 2
+updates:
+    - package-ecosystem: pip
```

Comment:
I think `npm` is the one this has been targeting the most, right?

Reply:
I can add npm but I dont think we use react/ node.js anywhere in gem5. I think so far all dependabot PRs have been python library updates.

Reply:
My bad. You're right. I thought that the gem5-resources-manager was written with node.